website/registrations/models/registration.py

```diff
@@ -59,6 +59,16 @@ class Meta:
     has_problems_with_signing_an_nda = models.BooleanField(default=False)
     comments = models.TextField(null=True, blank=True)
 
+    @property
+    def project(self):
+        """Get the project of a registration."""
+        return self.projects.first()
+
+    @project.setter
+    def project(self, value):
+        """Set the project of a registration."""
+        self.projects.set([value])
```

Comment:
Shouldn't we append here?

Reply:
Hmmm I dont think so, since for most cases (engineers) we want people to just be in 1 project